test-data/unit/lib-stub/types.pyi

```diff
@@ -1,4 +1,34 @@
-from typing import TypeVar
+from typing import TypeVar, Optional, List, Any, Generic, Sequence
 T = TypeVar('T')
+
 def coroutine(func: T) -> T:
     return func
+
+class bool: ...
+
+class ModuleSpec:
```

Comment:
Is it possible to prune the contents of this file some? It looks like you just copied from typeshed -- but in order to avoid slowing down the tests we should really only include attributes that tests reference.